pkgs/development/python-modules/aiohttp-swagger/default.nix

```
@@ -14,40 +15,33 @@
 buildPythonPackage rec {
   pname = "aiohttp-swagger";
   version = "1.0.15";
-  format = "setuptools";
+  pyproject = true;
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "cr0hn";
-    repo = pname;
-    rev = version;
+    repo = "aiohttp-swagger";
+    tag = version;
     hash = "sha256-M43sNpbXWXFRTd549cZhvhO35nBB6OH+ki36BzSk87Q=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
     aiohttp
     jinja2
     markupsafe
     pyyaml
   ];
 
-  nativeCheckInputs = [
-    pytestCheckHook
-    pytest-aiohttp
+  pythonRelaxDeps = [
+    "markupsafe"
+    "jinja2"
   ];
 
-  postPatch = ''
-    substituteInPlace requirements.txt \
-      --replace "markupsafe~=1.1.1" "markupsafe>=1.1.1" \
-      --replace "jinja2~=2.11.2" "jinja2>=2.11.2"
-  '';
-
-  preCheck = ''
-    # The custom client is obsolete
-    rm tests/conftest.py
-  '';
-
   pythonImportsCheck = [ "aiohttp_swagger" ];
 
   meta = with lib; {
```

```suggestion
  meta = {
```

nit: it's good to clean this up too.